R/count_teleconnections.R

```diff
@@ -123,6 +132,11 @@ count_watershed_teleconnections <- function(data_dir,
             }else{
               tc_n_landclasses = number_landclasses_ex_ag
             }
+        # TELECONNECTION - NUMBER OF FLOOD CONTROL DAMS WITHIN WATERSHED.
+        flood_control_sp <- sf::as_Spatial(flood_control_dams)
```

Comment:
add `importFrom sf as_Spatial` to top of the script and remove `sf::` (unless `as_Spatial` conflicts with another package)

Reply:
I had that originally but even with it at the top of the page it was still having trouble reading 'as_Spatial' . When I added the 'sf::' to the beginning it fixed that, so I'm assuming it may conflict with another package.

Reply:
Interesting... perhaps try adding `as` in addition to `as_Spatial`... if there's still an issue then we can certainly live with `sf::` !!